pkg/docker/docker.go

```diff
@@ -483,9 +547,10 @@ func runContainerTar(opts RunContainerOptions, config docker.Config, imageMetada
 			cmd = []string{"/bin/sh", "-c", opts.CommandOverrides(strings.Join(cmd[2:], " "))}
 		}
 	}
+	// TODO this message is misleading, as we're not running any container
+	// yet, we have just determined a value for cmd...
```

Comment:
@bparees @mfojtik would you like us to update this message below to something more precise? If so, suggestions on what should it be are welcome.